src/Integrations/Integrations/CodeIgniter/V2/CodeIgniterIntegration.PHP5.php

```diff
@@ -64,11 +64,6 @@ public function registerIntegration(\CI_Router $router, SpanData $rootSpan, $ser
         $rootSpan->meta[Tag::COMPONENT] = CodeIgniterIntegration::NAME;
         $rootSpan->meta[Tag::SPAN_KIND] = 'server';
 
-        if ('cli' !== PHP_SAPI) {
-            $normalizedPath = \DDTrace\Util\Normalizer::uriNormalizeincomingPath($_SERVER['REQUEST_URI']);
-            $rootSpan->resource = "{$_SERVER['REQUEST_METHOD']} $normalizedPath";
```

Comment:
Just, why are we removing these in CodeIgniter and Wordpress?

Reply:
yes, exactly :-)